selection_tools/include/Particle.hpp

```diff
@@ -29,8 +29,12 @@ class Particle
   double eta();
   double momentum() const;
   double pt() const;
-  
-  
+  double Px() const;
+  double M() const;
+  double E() const;
+  TVector3 boost() const;  
+  Particle* transform(TVector3 boost);
```

Comment:
And then this could be called "get_boosted_particle" or something like this. Or better - it could be a copy constructor - you give it a particle and boost and it returns a new, boosted particle.